File: Documentation/3.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/HowTo/Clear.cs
using Raven.Client.Document;
using Raven.Documentation.CodeSamples.Orders;

namespace Raven.Documentation.CodeSamples.ClientApi.Session.HowTo
{
	public class Clear
	{
		private interface IFoo
		{
			#region clear_1
			void Clear();
			#endregion
		}

		public Clear()
		{
			using (var store = new DocumentStore())
			{
				using (var session = store.OpenSession())
				{
					#region clear_2
					session.Store(new Employee
									  {
										  FirstName = "John",
										  LastName = "Doe"
									  });

					session.Advanced.Clear();

					session.SaveChanges(); // nothing will happen
					#endregion
				}
			}
		}
	}
}

File: Documentation/3.0/Samples/java/src/test/java/net/ravendb/clientapi/session/howto/Clear.java
package net.ravendb.clientapi.session.howto;

import net.ravendb.client.document.DocumentStore;
import net.ravendb.client.IDocumentSession;
import net.ravendb.client.IDocumentStore;
import net.ravendb.samples.northwind.Employee;

public class Clear {

  @SuppressWarnings("unused")
  private interface IFoo {
    //region clear_1
    public void clear();
    //endregion
  }

  public Clear() throws Exception {
    try (IDocumentStore store = new DocumentStore()) {
      try (IDocumentSession session = store.openSession()) {
        //region clear_2
        Employee employee = new Employee();
        employee.setFirstName("John");
        employee.setLastName("Doe");

        session.store(employee);

        session.advanced().clear();

        session.saveChanges(); // nothing will happen
        //endregion
      }
    }
  }
}
